Makefile

```
@@ -1,7 +1,16 @@
 #!make
 ENV_FILE ?= ./.env
 include $(ENV_FILE)
-DOCKER_COMPOSE :=  docker-compose -f docker-compose.yml -f ./conf/camino/$(COMPOSE_FILE) --env-file=$(ENV_FILE)
+ifndef BASE_COMPOSE_FILE
+override BASE_COMPOSE_FILE = docker-compose.core.cms.yml
+endif
+ifndef COMPOSE_FILE
+override COMPOSE_FILE_OVERRIDE =
+else
+override COMPOSE_FILE_OVERRIDE =  ${COMPOSE_FILE}
+endif
+
```

Should we add a default for `CAMINO_HOME` here for full backwards compatibility? Like 
```suggestion
ifndef CAMINO_HOME
override CAMINO_HOME := $(lastword $(abspath $(MAKEFILE_LIST)))
endif
```

(not tested, but found [here](https://stackoverflow.com/questions/18136918/how-to-get-current-relative-directory-of-your-makefile#comment116644351_18137056))